recipes/pyaerocom/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pyaerocom" %}
+{% set version = "0.8.0rc1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8fcf2cf511cc11b3ff4bee81d742a0c8d67c8e5617590be844fbfbe769193a05
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python >=3.5
```

If it's not `noarch: python`
```suggestion
    - python
```